common/error.h

```diff
@@ -96,14 +96,21 @@ class Error {
 	/**
 	 * Construct a new Error with the specified error code and an augmented
 	 * error message. Specifically, the provided extra text is suitably
-	 * appended to the default message.
+	 * appended to the default message, used for English messages.
 	 */
 	Error(ErrorCode code, const String &extra);
 
+	/**
+	 * Construct a new Error with the specified error code and an augmented
+	 * error message. Specifically, the provided extra text is suitably
+	 * appended to the default message, used for localized messages.
+	 */
+	Error(ErrorCode code, const U32String &extra);
```

Comment:
Do we have cases were we need to create an `Error` with a localised message (as opposed toan english message that is then localised using our Translation Manager)?